homeassistant/components/fibaro/lock.py

```diff
@@ -31,27 +33,21 @@ async def async_setup_entry(
 class FibaroLock(FibaroDevice, LockEntity):
     """Representation of a Fibaro Lock."""
 
-    def __init__(self, fibaro_device):
+    def __init__(self, fibaro_device: Any) -> None:
```

Comment:
Is it possible to use a more narrow type hint than `Any`?

Reply:
Unfortunately not, that is an object created by the used library "fiblary3" and is a mix of deserialized JSON customized with some injected methods. So to improve that code the underlaying library needs to be improved first.

Reply:
Is it not possible to reference that library object (whatever it is?)

Is it here?:
https://github.com/pbalogh77/fiblary/blob/e332289320857c837b7e68f8d969e2338280116c/fiblary3/client/v4/client.py#L47

Reply:
I am guessing that until progress is made on #71542 it may be difficult to improve on this.
cc @graham33

Reply:
I think as well that here `Any` is probably the best we could do at the moment.
But probably using `fiblary3.client.v4.models.DeviceModel | fiblary3.client.v4.models.SceneModel` would also be a valid typing, but because this classes do not have much predefined attributes / functions this is more or less equals as using `Any`.

Reply:
That sounds way better than `Any`.

Reply:
I changed the typing as suggested.